webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableFormats.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+export const hostsCountFormat = (value, { rowData: { name } }) => (
+  <a href={`hosts?search=model+%3D+"${encodeURI(name)}"`}>{value}</a>
+);
+
+export const actionsFormat = (value, { rowData: { name, id } }) =>
+  (value.includes('delete') ? (
+    <span className="btn btn-sm btn-default">
```

Comment:
:+1: to handle deleting via modal later on, but maybe we should use patternfly button component?

Reply:
Yes, this part is going to be done soon. `Katello` has `ConfirmDialog` that can be used here. 
I just didn't want to overload the number of changed files.

Reply:
thats fine, we should probably replace all delete buttons across vs just change it here.